doc/api/tls.markdown

```diff
@@ -231,9 +231,9 @@ Typical flow:
 5. Client validates the response and either destroys socket or performs a
    handshake.
 
-NOTE: `issuer` could be null, if the certificate is self-signed or if the issuer
-is not in the root certificates list. (You could provide an issuer via `ca`
-option.)
+NOTE: `issuer` could be `null`, if the certificate is self-signed or if the
+issuer is not in the root certificates list. (You could provide an issuer via
```

Comment:
Nit, while you're in there: Remove personal pronoun. 

Before:

> You could provide an issuer via `ca` option.

After, something like:

> An issuer may be provided via the `ca` option.